Index: certificate-authority/ca/bin/fsinf-ca-sign.sh
===================================================================
--- certificate-authority.orig/ca/bin/fsinf-ca-sign.sh	2013-08-11 11:04:50.902610869 +0200
+++ certificate-authority/ca/bin/fsinf-ca-sign.sh	2013-08-11 11:07:50.594611009 +0200
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-python manage.py sign $@
+python /usr/share/fsinf-certificate-authority/manage.py sign $@
Index: certificate-authority/ca/bin/fsinf-ca-list.sh
===================================================================
--- certificate-authority.orig/ca/bin/fsinf-ca-list.sh	2013-08-11 11:04:50.882611126 +0200
+++ certificate-authority/ca/bin/fsinf-ca-list.sh	2013-08-11 11:08:11.462610966 +0200
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-python manage.py list $@
+python /usr/share/fsinf-certificate-authority/manage.py list $@
Index: certificate-authority/ca/bin/fsinf-ca-remove.sh
===================================================================
--- certificate-authority.orig/ca/bin/fsinf-ca-remove.sh	2013-08-11 11:04:50.882611126 +0200
+++ certificate-authority/ca/bin/fsinf-ca-remove.sh	2013-08-11 11:08:21.166615103 +0200
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-python manage.py remove $@
+python /usr/share/fsinf-certificate-authority/manage.py remove $@
Index: certificate-authority/ca/bin/fsinf-ca-view.sh
===================================================================
--- certificate-authority.orig/ca/bin/fsinf-ca-view.sh	2013-08-11 11:04:50.882611126 +0200
+++ certificate-authority/ca/bin/fsinf-ca-view.sh	2013-08-11 11:08:28.378612747 +0200
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-python manage.py view $@
+python /usr/share/fsinf-certificate-authority/manage.py view $@
Index: certificate-authority/ca/bin/fsinf-ca-watch.sh
===================================================================
--- certificate-authority.orig/ca/bin/fsinf-ca-watch.sh	2013-08-11 11:04:50.882611126 +0200
+++ certificate-authority/ca/bin/fsinf-ca-watch.sh	2013-08-11 11:08:37.786609342 +0200
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-python manage.py watch $@
+python /usr/share/fsinf-certificate-authority/manage.py watch $@
